apps/files/lib/Command/TransferOwnership.php

```diff
@@ -87,6 +103,31 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			return self::FAILURE;
 		}
 
+		$path = ltrim($input->getOption('path'), '/');
+		$includeExternalStorage = $input->getOption('include-external-storage');
+		if ($includeExternalStorage) {
+			$mounts = $this->mountManager->findIn('/' . rtrim($sourceUserObject->getUID() . '/files/' . $path, '/'));
+			/** @var IMountPoint[] $mounts */
+			$mounts = array_filter($mounts, fn ($mount) => $mount->getMountProvider() === ConfigAdapter::class);
+			if (count($mounts) > 0) {
+				$output->writeln(count($mounts) . ' external storages will be transferred:');
+				foreach ($mounts as $mount) {
+					$output->writeln('  - <info>' . $mount->getMountPoint() . '</info>');
+				}
+				$output->writeln('');
+				$output->writeln('<error>Any other users with access to these external storages will loose access to the files.</error>');
```

Comment:
Even though it is an important message, it is not an error.

```suggestion
				$output->writeln('<warning>Any other users with access to these external storages will loose access to the files.</warning>');
```